data/migrations/20190501115750_partners.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable('partners', table => {
+		table
+			.integer('manager_id')
+			.unsigned()
+			.notNullable()
+			.references('users.user_id');
+
+		table
+			.integer('cleaner_id')
+			.unsigned()
+			.notNullable()
+			.references('users.user_id');
+
+		table.unique(['manager_id', 'cleaner_id']);
```

Comment:
Cool, this basically makes the connection between the manager_id and cleaner_id unique.

It doesn't make only each unique in it's respective column.